pom.xml

```diff
@@ -79,12 +79,12 @@
             <dependency>
                 <groupId>org.apache.logging.log4j</groupId>
                 <artifactId>log4j-api</artifactId>
-                <version>2.16.0</version>
+                <version>2.23.1</version>
             </dependency>
             <dependency>
                 <groupId>org.apache.logging.log4j</groupId>
                 <artifactId>log4j-core</artifactId>
-                <version>2.16.0</version>
+                <version>2.23.1</version>
```

Comment:
We're downgrading these?

Reply:
No this is an upgrade: https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-core

Reply:
Sorry it seems I have forgotten how numbers are ordered.